Formula/s/spicetify-cli.rb

```diff
@@ -0,0 +1,46 @@
+class SpicetifyCli < Formula
+  desc "Command-line tool to customize Spotify client"
+  homepage "https://github.com/spicetify/spicetify-cli"
+  url "https://github.com/spicetify/spicetify-cli/archive/refs/tags/v2.27.1/v2.27.1.tar.gz"
+  sha256 "3e41c4ac65d916fea662037a492ad63490a22255fad7397731f8fb6efcb3cf0d"
+  license "LGPL-2.1-only"
+  head "https://github.com/spicetify/spicetify-cli.git", branch: "master"
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = %W[
+      -s -w
+      -X main.version=#{version}
+    ]
+    system "go", "build", *std_go_args(ldflags: ldflags, output: libexec/"spicetify")
+    cd buildpath do
+      libexec.install [
+        "css-map.json",
+        "CustomApps",
+        "Extensions",
+        "globals.d.ts",
+        "jsHelper",
+        "Themes",
+      ]
+      bin.install_symlink libexec/"spicetify"
+    end
+  end
+
+  test do
+    spotify_folder = testpath/"com.spotify.Client"
+    pref_file = spotify_folder/"com.spotify.client.plist"
+    mkdir_p spotify_folder
+    touch pref_file
```

Comment:
```suggestion
```

Reply:
@chenrui333, the files need to exist otherwise `spicetify` will error out.

Reply:
I see, sounds good then.